src/pages/wishlist/wishlist.jsx

```diff
@@ -0,0 +1,93 @@
+                            </th>
+                            <th className="py-3 hidden md:table-cell"></th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        {wishListItems?.map((item) => (
+                            <>
+                                <tr className="hidden md:table-row">
+                                    <td className="w-4/5 border hover:bg-slate-300/20">
+                                        <div className="flex items-center gap-x-10 w-full p-2">
+                                            <Link
+                                                to={
+                                                    "/productInfo/" + item.id
+                                                }
+                                            >
+                                                <img
+                                                    src={item?.imageUrl}
+                                                    className="w-36 object-cover hover:opacity-80"
+                                                    alt="404 not found"
+                                                />
+                                            </Link>
+                                            <div className="font-bold">
+                                                <Link
+                                                    to={
+                                                        "/productInfo/" + item.id
+                                                    }
+                                                >
+                                                    <p className="text-gray-900 hover:text-[#5e3fde]">
+                                                        {item?.title}
+                                                    </p>
+                                                </Link>
+                                                <p className="text-gray-900">
+                                                    RS. {item.price}
+                                                </p>
+                                            </div>
+                                        </div>
+                                    </td>
+                                    <td className="p-5 border text-center hover:bg-slate-300/20">
+                                        <button
+                                            onClick={() => handleRemoveFromWishlist(item)}
+                                        >
+                                            <ZmdiDelete/>
+                                        </button>
+                                    </td>
+                                </tr>
+                            </>
+                        ))}
+
+                        {
+                            !wishListItems?.length && (
+                                <tr>
+                                    <td>No Items here..</td>
```

Comment:
display statement like
- "Oops! Your wishlist looks empty. Browse and add items to save for later."
- "You haven't added any items to your wishlist. Discover something special today!"
- "Nothing here yet! Explore our collection and add items to your wishlist."